docs/next-release-notes.md

```diff
@@ -0,0 +1,341 @@
+# New features
+
+## Updated Kafka support
+
+Updated the Apache Kafka core dependency to version 3.3.1.
+
+https://github.com/apache/druid/pull/13176
+
+## Query engine
+
+### BIG_SUM SQL function
+
+Added SQL function `BIG_SUM` that uses the [Compressed Big Decimal](https://github.com/apache/druid/pull/10705) Druid extension.
+
+https://github.com/apache/druid/pull/13102
+
+### Added Compressed Big Decimal min and max functions
+
+Added min and max functions for Compressed Big Decimal and exposed these functions via SQL: BIG_MIN and BIG_MAX.
+
+https://github.com/apache/druid/pull/13141
+
+### Metrics used to downsample bucket
+
+Changed the way the MSQ task engine determines whether or not to downsample data, to improve accuracy. The task engine now uses the number of bytes instead of number of keys.
+
+https://github.com/apache/druid/pull/12998
+
+### MSQ Docker improvement
+
+Enabled MSQ query engine for Docker by default.
+
+https://github.com/apache/druid/pull/13069
+
+### Improved MSQ warnings
+
+For disallowed MSQ warnings of certain types, the warning is now surfaced as the error.
+
+https://github.com/apache/druid/pull/13198
+
+### Added support for indexSpec
+
+The MSQ task engine now supports the `indexSpec` context parameter.
+
+https://github.com/apache/druid/pull/13275
+
+### Added task start status to the worker report
+
+Added `pendingTasks` and `runningTasks` fields to the worker report for the MSQ task engine.
+
+https://github.com/apache/druid/pull/13263
```

Comment:
This should also mention https://github.com/apache/druid/pull/13291 which is the UI followup to the same work.

Reply:
Although I guess it is mentioned in the web console section... maybe these sections should link to each other :-p